content/08-mongodb/08-mongodb-datatypes.mdx

```diff
@@ -619,3 +619,67 @@ It is important to know what data types are available to you when using a databa
 
 If you are interested in checking out Prisma with a MongoDB database, you can check out the [data connector documentation](https://www.prisma.io/docs/concepts/database-connectors/mongodb).
 
+## FAQ
+
+<details><summary>Can you store regular expression patterns in MongoDB?</summary>
+
+Yes, MongoDB allows for the storage of regular expression with the [`Regular Expression` data type](https://www.prisma.io/dataguide/mongodb/mongodb-datatypes#regular-expression).
+
+For instance, you can insert the `Regular Expression` data type like this:
+
+```sql
+db.mytestcoll.insertOne({exampleregex: /tt/})
+{
+        "acknowledged": true,
+        "insertedId": ObjectId("614b37296a124db40ae74d16")
+}
+db.mytestcoll.insertOne({exampleregext:/t+/})
+{
+        "acknowledged": true,
+        "insertedId": ObjectId("614b37296a124db40ae74d17")
+}
+```
+
+</details>
+
+<details><summary>How are dates represented as a data type in MongoDB?</summary>
```

Comment:
Do we have anything we could say about the internal representation?  I think the info you have here is good, but it might also be good to say something about how it's tracked internally as an integer representing the time since the [Unix Epoch](https://en.wikipedia.org/wiki/Unix_time).

https://docs.mongodb.com/manual/reference/bson-types/#std-label-document-bson-type-date